images/base/Dockerfile

```diff
@@ -81,10 +81,10 @@ RUN curl -fsSL "https://download.docker.com/linux/$(. /etc/os-release; echo "$ID
 
 # Install CNI binaries to /opt/cni/bin
 # TODO(bentheelder): doc why / what here
-ARG CNI_VERSION="0.6.0"
-ARG CNI_TARBALL="cni-plugins-${ARCH}-v${CNI_VERSION}.tgz"
-ARG CNI_BASE_URL="https://storage.googleapis.com/kubernetes-release/network-plugins/"
-ARG CNI_URL="${CNI_BASE_URL}${CNI_TARBALL}"
+ARG CNI_VERSION="v0.7.4"
```

Comment:
probably best to bump this only after the official packages are bumped to this version:
https://github.com/kubernetes/release/blob/4de61a4d8b5fdf2a9a5aac4013afcbc94b54b2ad/debian/xenial/kubernetes-cni/debian/rules

is it a requirement for ipv6 to work?

Reply:
it's an option, you can read more about it here https://github.com/containernetworking/cni/issues/531

I think that it can be solved within `kind` too